### DISCUSSION TITLE
Beginer setup

### DISCUSSION THREAD

**Discussion by DunaSpice** [MrXandbadas/MrX_OAI_Assistant_Manager#12] [Q&A] [answered]
I am having trouble setting up and running the script, if you could provide some detail I would be glad to later post them in an organised way. I set up API_KEY and ORG_ID installed pyautogen BUT FOR the life of me can not understand how to proceed and run the script. The example_cmd_chat.py is not in the repository. Do I create it? I am lost(

**Comment by MrXandbadas** [comment, marked as answer]
So sorry for the confusion the example chat is "CMD_Agent_Chat.py"
I've updated the readme accordingly!

*facepalm* on my behalf
